tests/functions/test_map_to_s3.py

```diff
@@ -8,6 +8,24 @@
 CONF_PATH = "cdn_lambda/functions/map_to_s3/map_to_s3.json"
 
 
+@pytest.mark.parametrize(
+    "uri",
+    [
+        "www.example.com/content/origin/rpms/repo/ver/filename",
+        "www.example.com/origin/rpm/repo/ver/filename",
+        "www.example.com/content/origin/rpm/repo/ver/filename",
+    ],
+    ids=[
+        "content/origin -> origin",
+        "origin/rpm -> origin/rpms",
+        "content/origin/rpm -> origin/rpms",
+    ],
+)
+def test_map_to_s3_uri_alias(uri):
+    expected = "www.example.com/origin/rpms/repo/ver/filename"
+    assert LambdaClient(conf_file=CONF_PATH).uri_alias(uri) == expected
```

Comment:
This test is OK but it doesn't really seem sufficient.  Wouldn't the test still pass even if you completely forgot to call uri_alias from within handler?  (Or if someone accidentally removed it during a refactor.)

In my opinion, there should be at least one test which covers this behavior from the top level of the handler.

Reply:
Done.